README.md

```diff
@@ -170,18 +176,23 @@ decompressed_values = decompress(compressed_values)
 print("Decompression successful. Decompressed data length: ", len(decompressed_values))
 ```
 
-TerseTS provides `./bindings/python/tersets/__init__.py` as binding for Python which can be imported directly into a Python program with `import tersets`. The binding automatically loads the native library but assumes it is not moved.
+TerseTS provides Python bindings located in `./bindings/python/tersets/__init__.py`, which can be directly imported into a Python program using `import tersets`. To install the bindings, navigate to the Python binding root directory and run `pip install .` as described in the [Python bindings README](bindings/python/README.md). The bindings automatically load the native library, assuming it remains in its default location.
 
-- **`compress()` Function:**
+
+- **`compress(values, method, configuration)` Function:**
    - **Parameters:**
-      - `uncompressed_values`: The array of values to compress.
-      - `method`: The Python binding provides the `Method` enum to provide direct access to the available methods supported by `TerseTS`. The supported compression methods are specified in `src/tersets.zig`.
-      - `error_bound`: The error bound.
-   - **Returns:** A list of compressed values.
-- **`decompress()` Function:**
+      - `values`: A list, tuple, or `numpy.ndarray` of floats representing the data to compress.
```

Comment:
Must it be one of these specific types or is any Python [Sequence](https://docs.python.org/3/glossary.html#term-sequence) OK?